README.md

```diff
@@ -383,6 +382,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/samuelard7"><img src="https://avatars.githubusercontent.com/u/98638849?v=4?s=100" width="100px;" alt="Richard Samuel"/><br /><sub><b>Richard Samuel</b></sub></a><br /><a href="#a11y-samuelard7" title="Accessibility">️️️️♿️</a> <a href="#question-samuelard7" title="Answering Questions">💬</a> <a href="#audio-samuelard7" title="Audio">🔊</a> <a href="#blog-samuelard7" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Asamuelard7" title="Bug reports">🐛</a> <a href="#business-samuelard7" title="Business development">💼</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=samuelard7" title="Code">💻</a> <a href="#content-samuelard7" title="Content">🖋</a> <a href="#data-samuelard7" title="Data">🔣</a> <a href="#design-samuelard7" title="Design">🎨</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=samuelard7" title="Documentation">📖</a> <a href="#eventOrganizing-samuelard7" title="Event Organizing">📋</a> <a href="#example-samuelard7" title="Examples">💡</a> <a href="#financial-samuelard7" title="Financial">💵</a> <a href="#fundingFinding-samuelard7" title="Funding Finding">🔍</a> <a href="#ideas-samuelard7" title="Ideas, Planning, & Feedback">🤔</a> <a href="#infra-samuelard7" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#maintenance-samuelard7" title="Maintenance">🚧</a> <a href="#mentoring-samuelard7" title="Mentoring">🧑‍🏫</a> <a href="#platform-samuelard7" title="Packaging/porting to new platform">📦</a> <a href="#plugin-samuelard7" title="Plugin/utility libraries">🔌</a> <a href="#projectManagement-samuelard7" title="Project Management">📆</a> <a href="#research-samuelard7" title="Research">🔬</a> <a href="https://github.com/TejasQ/opensauced-guestbook/pulls?q=is%3Apr+reviewed-by%3Asamuelard7" title="Reviewed Pull Requests">👀</a> <a href="#security-samuelard7" title="Security">🛡️</a> <a href="#talk-samuelard7" title="Talks">📢</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=samuelard7" title="Tests">⚠️</a> <a href="#tool-samuelard7" title="Tools">🔧</a> <a href="#translation-samuelard7" title="Translation">🌍</a> <a href="#tutorial-samuelard7" title="Tutorials">✅</a> <a href="#userTesting-samuelard7" title="User Testing">📓</a> <a href="#video-samuelard7" title="Videos">📹</a> <a href="#promotion-samuelard7" title="Promotion">📣</a></td>
+    </tr>
+
```

Comment:
There are a couple of things here:

- Looking at the contributors' table at README, I believe, the profile after yours should be within the `<tr></tr>` tags.
- There is a closing `</tr>` tag on line 386 and 389, but there is no opening `<tr>` tag in between these lines.